python/ct/client/db_reporter.py

```diff
@@ -19,23 +19,25 @@ def __init__(self, cert_db, log_key, checks=all_checks.ALL_CHECKS):
         self._writer = None
         super(CertDBCertificateReport, self).__init__(checks=checks)
 
+    def _writer_ready(self):
```

Comment:
This function extraction definitely helps the readability of the code below particular in the negated case.